packages/inventory/src/containers/ProfileView/index.tsx

```diff
@@ -0,0 +1,272 @@
+import React, { useEffect, useState } from 'react';
+import { Row, Col, Button, Table } from 'antd';
+import { OpenSRPService } from '@opensrp/react-utils';
+import {
+  TreeNode,
+  hierarchyReducer,
+  hierarchyReducerName,
+  locationUnitsReducer,
+  fetchLocationUnits,
+  locationUnitsReducerName,
+  LocationUnit,
+  loadJurisdictions,
+  getLocationsBySearch,
+  getTreesByIds,
+} from '@opensrp/location-management';
+import { connect, useDispatch, useSelector } from 'react-redux';
+import { ColumnsType } from 'antd/lib/table/interface';
+import { columns } from './utils';
+import { sendErrorNotification } from '@opensrp/notifications';
+import { Spin } from 'antd';
+import { Link, RouteComponentProps, useParams } from 'react-router-dom';
+import reducerRegistry from '@onaio/redux-reducer-registry';
+import { BrokenPage, useHandleBrokenPage } from '@opensrp/react-utils';
+import { Helmet } from 'react-helmet';
+import {
+  GET_INVENTORY_BY_SERVICE_POINT,
+  INVENTORY_SERVICE_POINT_LIST_VIEW,
+  GEOGRAPHIC_LEVEL,
+  LOCATIONS_GET_ALL_SYNC_ENDPOINT,
+  INVENTORY_SERVICE_POINT_PROFILE_PARAM,
+} from '../../constants';
+import { CommonProps, defaultCommonProps } from '../../helpers/common';
+import {
+  ADD_NEW_INVENTORY_ITEM,
+  SERVICE_POINT_INVENTORY,
+  EDIT_SERVICE_POINT,
+  INVENTORY_ITEMS,
+  ERROR_OCCURRED,
+  REGION_LABEL,
+  DISTRICT_LABEL,
+  TYPE_LABEL,
+  LAT_LONG_LABEL,
+  COMMUNE_LABEL,
+  SERVICE_POINT_ID_LABEL,
+  BACK_TO_SERVICE_POINT_LIST,
+} from '../../lang';
+import '../../index.css';
+import {
+  fetchInventories,
+  getInventoriesByExpiry,
+  inventoryReducer,
+  inventoryReducerName,
+  Inventory,
+} from '../../ducks/inventory';
+import { getNodePath } from './utils';
+import { ServiceType } from '@opensrp/location-management/src/ducks/location-units';
```

Comment:
The type for ServiceType has changed  in https://github.com/OpenSRP/web/pull/441/files

Reply:
Let's export the type in location-management `index.ts` so that we can do `fimport {ServiceType} from @opensrp/location-management`